bin/animation_assets/manifest_builder.rb

```diff
@@ -74,10 +74,10 @@ def rebuild_animation_library_manifest
     EOS
 
     if @options[:spritelab] && @options[:upload_to_s3]
-      info "Uploading manifests/spritelabCostumeLibrary.json to S3"
+      info "Uploading animation-manifests/manifests/spritelabCostumeLibrary.json to S3"
       AWS::S3.upload_to_bucket(
         DEFAULT_S3_BUCKET,
-        "manifests/spritelabCostumeLibrary.json",
+        "animation-manifests/manifests/spritelabCostumeLibrary.json",
```

Comment:
nit: move this and the localized version to a constant and a function, respectively